src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2258,38 +2258,13 @@ protected int toInt(String string, int defaultNumber) {
     }
 
     protected int getIntegerValue(String string, int defaultNumber) {
-
-        int number = defaultNumber;
-
-        if (string != null && string.length() > 0) {
-            number = Integer.parseInt(string.trim());
-        }
-
-        return number;
+        // TODO REFACTOR replace getIntegerValue with StringUtilities.getIntegerValue
+        return StringUtilities.getIntegerValue(string, defaultNumber);
```

Comment:
seems that TODO todone?   so get rid of the TODO

Reply:
willd o

Reply:
pushed fix